### PR TITLE
[CARBONDATA-1135] Added partition column info in describe formatted

### PR DESCRIPTION
**Pull request by kunal642**
Currently the partition columns are not listed in describe formatted command.

This PR will enable the listing of the same

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2238/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/111/<h2>Failed Tests: <span class='status-failure'>5</span></h2><h3><a name='carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-core' /><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-core/testReport'>carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-core</a>: <span class='status-failure'>2</span></h3><ul><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-core/testReport/org.apache.carbondata.core.writer/CarbonFooterWriterTest/testWriteFactMetadata/'><strong>org.apache.carbondata.core.writer.CarbonFooterWriterTest.testWriteFactMetadata</strong></a></li><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-core/testReport/org.apache.carbondata.core.writer/CarbonFooterWriterTest/testReadFactMetadata/'><strong>org.apache.carbondata.core.writer.CarbonFooterWriterTest.testReadFactMetadata</strong></a></li></ul><h3><a name='carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark' /><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-spark/testReport'>carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-spark/testReport/org.apache.carbondata.spark.testsuite.createtable/TestCreateTableSyntax/describe_formatted_on_hive_table_and_carbon_table/'><strong>org.apache.carbondata.spark.testsuite.createtable.TestCreateTableSyntax.describe formatted on hive table and carbon table</strong></a></li></ul><h3><a name='carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark-common-test' /><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-spark-common-test/testReport'>carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark-common-test</a>: <span class='status-failure'>2</span></h3><ul><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-spark-common-test/testReport/org.apache.carbondata.spark.testsuite.describeTable/TestDescribeTable/test_describe_formatted_table/'><strong>org.apache.carbondata.spark.testsuite.describeTable.TestDescribeTable.test describe formatted table</strong></a></li><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/111/org.apache.carbondata$carbondata-spark-common-test/testReport/org.apache.carbondata.spark.testsuite.partition/TestDDLForPartitionTable/test_describe_formatted_for_partition_column/'><strong>org.apache.carbondata.spark.testsuite.partition.TestDDLForPartitionTable.test describe formatted for partition column</strong></a></li></ul>

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/206/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2328/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/207/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/205/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2329/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2330/

File: integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestDDLForPartitionTable.scala [hunk 1]
```diff
@@ -130,6 +138,7 @@ class TestDDLForPartitionTable  extends QueryTest with BeforeAndAfterAll {
     sql("drop table if exists hashTable")
     sql("drop table if exists rangeTable")
     sql("drop table if exists listTable")
+    sql("drop table test")
```
Comment (gvramana) [inline integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestDDLForPartitionTable.scala]: Use drop table if exists

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2379/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/258/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark-common-test' /><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/258/org.apache.carbondata$carbondata-spark-common-test/testReport'>carbondata-pr-spark-1.6/org.apache.carbondata:carbondata-spark-common-test</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://builds.apache.org/job/carbondata-pr-spark-1.6/258/org.apache.carbondata$carbondata-spark-common-test/testReport/org.apache.carbondata.spark.testsuite.partition/TestDDLForPartitionTable/test_describe_formatted_for_partition_column/'><strong>org.apache.carbondata.spark.testsuite.partition.TestDDLForPartitionTable.test describe formatted for partition column</strong></a></li></ul>

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2386/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/265/

**Comment by gvramana** [comment]
LGTM

**Pull request closed without merging**